changelog/11850.improvement.rst

```diff
@@ -0,0 +1 @@
+Added the new `sys.last_exc` value to `pytest_runtest_call()` in `src/_pytest/runner.py`.
```

Comment:
Since the changelog is intended for end-users to read, they don't really need to know about details like `pytest_runtest_call` or the internal file it's done in. I suggest something like this:

```suggestion
Added support for :data:`sys.last_exc` for post-mortem debugging on Python>=3.12.
```